_topic_map.yml

```diff
@@ -622,6 +622,10 @@ Topics:
   Topics:
   - Name: Getting started with Helm on OpenShift Container Platform
     File: getting-started-with-helm-on-openshift-container-platform
+    Distros: openshift-enterprise
+  - Name: Getting started with Helm on OKD
```

Comment:
@Preeticp Can you please review this update that touches Helm? It's to fix an instance of "OpenShift Container Platform" that was appearing in the TOC in OKD.

Preview:
* OKD: http://file.rdu.redhat.com/~ahoffer/2021/github-35357/github-35357/cli_reference/helm_cli/getting-started-with-helm-on-openshift-container-platform.html
* OCP: https://deploy-preview-35512--osdocs.netlify.app/openshift-enterprise/latest/cli_reference/helm_cli/getting-started-with-helm-on-openshift-container-platform.html

Reply:
Hi @bergerhoffer the previews look good to me. Thank you! 
However, does it make sense to remove 'OpenShift Container Platform' from there and maybe simply say OpenShift? WDYT? 
We are making changes to these sections soon, please let us know your thoughts so that we keep these nuances in mind going forward.

Reply:
@Preeticp thanks! So, we normally try to avoid just saying "OpenShift" and instead prefer the actual product names. However, I think that a good option might be to just remove the product reference all together. It could just be named "Getting started with Helm", because this whole doc set is all about doing xyz on OCP (or OKD). 

You could always just make this change in the name in the TOC too, and leave the product name in the assembly title as you have it now. If you want to go this route, I can always make that change (to remove the product name from the TOC title) in this PR instead of the workaround I've proposed here. Or I can go ahead with this and your team can consider it later. Just let me know how you want me to proceed!

Reply:
@bergerhoffer in the upcoming release for 4.9 we will be removing the getting started section and reorganizing the content  into Understanding, Installing, configuring and so on. One of our writers will submit a PR on it soon. Removing it from the TOC, as well as the assembly makes the most sense to me. 
How do you suggest we do this, 1. Remove it from the TOC through this PR and in our PRs we will remove it from the assembly as well? or 2. Should our PR address both concerns at once? Either approaches are Ok with us.

Reply:
@Preeticp I went ahead and removed the "on OCP" from both the title in the topic map and the assembly in this PR. This will get updated to all supported versions (4.6+).

Take a look and let me know if it looks okay to you!

Reply:
Looks perfect, thanks Andrea.